quiz175.ipynb

```diff
@@ -0,0 +1,154 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "bb0ecdce-72d2-40a4-918a-88cd2a21e541",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import csv"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "id": "210c6d31-482f-4da3-b537-8c502b5d4978",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "file = open(\"intro.txt\",'w',encoding='utf-8',newline='')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "id": "03a43046-1fe7-4ce2-9045-907cd0a920ae",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "l=[['My Name:Vrajkumar Vinodkumar Patel'],\n",
+    "   ['My Father Name:VinodKumar ThakorBhai Patel'],\n",
+    "   ['My Mother Name:MahesvariBen VinodKumar Patel'],\n",
+    "   ['Home Address:Main Road Near Post Office,Palsana,Surat'],\n",
+    "   ['My Birth Date:03\\10\\2004'],\n",
+    "   ['My favorite Game:Cricket'],\n",
+    "   ['My favorite Place:Somnath']]"
+   ]
+  },
```

Comment:
is it possible to take multiline input from the user. ?

Reply:
Yes,Using Loop we can take multiline  input from user.